ruby/neo4j/driver/internal/handlers/legacy_pull_all_response_handler.rb

```diff
@@ -65,7 +64,7 @@ def disable_auto_read_management
         end
 
         def peek_async
-          record = @records.first
+          record = @records.items.first
```

Comment:
this does not work, have a look at AutoPull...
this method needs to wait under some circumstances

Reply:
okay